Gravity/Gravity/DAL/RSAPI/RsapiDao.Get.cs

```diff
@@ -110,6 +110,43 @@ private T GetHydratedDTO<T>(RDO objectRdo, ObjectFieldsDepthLevel depthLevel)
 			return dto;
 		}
 
+		private void PopulateChoices(BaseDto dto, RDO objectRdo)
+		{
+			foreach ((PropertyInfo property, RelativityObjectFieldAttribute fieldAttribute) 
```

Comment:
I can't resolve lines 117 to 120. Getting two errors and have to rewrite the code in order to build:
Error 1: Foreach cannot operate on lambda expressions.
Error 2: Cannot specify constructor arguments in declaration.

Reply:
Are you using Visual Studio 2017?

Reply:
I was using VS2015...

Reply:
I have already. :)

Reply:
@mrobustelli needs to get on #18!